### PR TITLE
Use table UI component for workflow task index

### PR DESCRIPTION
**Pull request by gasman**
Follow-up to https://github.com/wagtail/wagtail/pull/10516

### CODE REVIEW

File: wagtail/admin/views/workflows.py [hunk 1]
```diff
@@ -382,6 +383,19 @@ def remove_workflow(request, page_pk, workflow_pk=None):
         return redirect("wagtailadmin_explore", page.id)
 
 
+class TaskTitleColumn(TitleColumn):
+    cell_template_name = "wagtailadmin/workflows/includes/task_title_cell.html"
+
+    def get_cell_context_data(self, instance, parent_context):
+        context = super().get_cell_context_data(instance, parent_context)
+        context["active"] = instance.active
+        return context
```
Comment (laymonage) [inline wagtail/admin/views/workflows.py:392]: It seems `instance` is available in the context (from the base `Column.get_cell_context_data()`), so we can do `instance.active` in the template and avoid the override

**Review by laymonage — COMMENTED** [review]
The workflow index view could use the same refactoring, is that for a separate PR or do you think we should tackle it here as well?

File: wagtail/admin/views/workflows.py (hunk 1 above)
Comment (gasman) [reply wagtail/admin/views/workflows.py]: Good spot! Updated and rebased now.

**Review by laymonage — APPROVED** [review]
Looks good to me, thanks! FYI the workflow index view could use the same treatment as well

**Comment by gasman** [comment]
Merged in cd5200c8e1ac0d7299fd9c398b2b994606b3c7d2

**Pull request closed without merging**